### PR TITLE
[material-ui-chip-input] Update to version 0.13.5

### PR DESCRIPTION
**Pull request by madvas**
**Extern:** The API did not change.

### CODE REVIEW

File: material-ui-chip-input/build.boot [hunk 1]
```diff
@@ -2,7 +2,7 @@
   :resource-paths #{"resources"}
   :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
```
Comment (Deraen) [inline material-ui-chip-input/build.boot]: This package should depend on`cljsjs/react` and `cljsjs/material-ui`.

Comment (madvas) [inline material-ui-chip-input/build.boot]: okay, fixed ;)

Comment (Deraen) [inline material-ui-chip-input/build.boot]: Hmm, I forgot that material-ui comes with its own React? If that is still the case, `cljsjs/react` should be dependency here.

Comment (madvas) [inline material-ui-chip-input/build.boot]: yep, fixed ;)

**Pull request merged**